src/main/java/org/hawkular/client/android/util/Preferences.java

```diff
@@ -40,7 +41,14 @@ private Keys() {
         public static final String BACKEND_ENVIRONMENT = "environment";
     }
 
-    private SharedPreferences serverPreferences;
+    public static final class Defaults {
+        private Defaults() {
+        }
+
+        public static final int BACKEND_PORT = Integer.MIN_VALUE;
```

Comment:
That looks odd .. is no port is given, will the user see a very negative number?
Why not use 18080 or such?